grader/actions.py

```diff
@@ -22,7 +22,7 @@
 from util.files import read_submission_file
 from util.http import get_json
 from util.personalized import user_personal_directory_path, select_generated_exercise_instance
-from util.shell import invoke_script, invoke_sandbox
+from util.shell import invoke_script
```

Comment:
I think I already told you to remove this whole file `grader/actions.py`. Aren't all these functions used solely with the chroot sandbox? At a quick glance, I still definitively think so.

Reply:
The function names here correspond to the action names used in the actual exercise config.yaml file (in the `actions` field). For example, they always tend to start with the `prepare` action.

Reply:
Yes, I tried to remove the `grader/actions.py` file, but it was causing me some trouble. Now, I removed it completely, and at least the course-template exercises are working fine.